src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest2.cs

```diff
@@ -1133,8 +1133,7 @@ public void BuildProguardEnabledProject (string rid)
 			if (!string.IsNullOrEmpty (rid)) {
 				proj.SetProperty ("RuntimeIdentifier", rid);
 			}
-			// FIXME: https://github.com/dotnet/msbuild/issues/11237, removed `(` and `)` characters
-			using (var b = CreateApkBuilder (Path.Combine ("temp", $"BuildProguard Enabled1{rid}"))) {
+			using (var b = CreateApkBuilder (Path.Combine ("temp", $"BuildProguard Enabled(1){rid}"))) {
```

Comment:
It appears the MSBuild fix is working:

![image](https://github.com/user-attachments/assets/aa414394-cd14-44d6-ae71-31f72c164819)